pcsx2/GS/Renderers/HW/GSTextureCache.cpp

```diff
@@ -414,7 +414,7 @@ GSTextureCache::Source* GSTextureCache::LookupSource(const GIFRegTEX0& TEX0, con
 	return src;
 }
 
-GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, const GSVector2i& size, int type, bool used, u32 fbmask, const bool is_frame, const int real_h)
+GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, const GSVector2i& size, int type, bool used, u32 fbmask, const bool is_frame, const int real_h, const int real_w)
```

Comment:
Could be worth swapping `real_h` and `real_w` here, having them around the other way in one function seems a bit inconsistent.

Reply:
Yeah, I'll do that today. I also thought I probably wanna do a std::max on the new height and new width, since before it was only checking if one changed, but right now it's possible it could shrink one direction when expanding the other, so I should probably avoid that :D